Changelog.md

```diff
@@ -13,6 +15,7 @@
 - Add optional callback queue parameter to reactive providers.
 
 - Bumped minimum version of ReactiveSwift to 2.0.
+- Add optional callback queue parameter to reactive providers.
```

Comment:
this is probably leftover from rebasing, but we have this entry already (line 15)